### PR TITLE
Add a short comment to some gapis RPC

### PR DESCRIPTION
**Pull request by hevrard**
Bug: N/A

### CODE REVIEW

File: gapis/service/service.proto [hunk 1]
```diff
@@ -731,6 +733,8 @@ service Gapid {
       returns (stream GetTimestampsResponse) {
   }
 
+  // ValidateDevice runs validation of the GPU driver on the requested
+  // device to check that GPU counters are within an expected range.
```
Comment (lpy) [inline gapis/service/service.proto:737]: to validate the GPU profiling capabilities of the requested device

Comment (hevrard) [reply gapis/service/service.proto]: done

Comment (johnplate) [reply gapis/service/service.proto]: Unfortunately now the line is too long :)

Comment (hevrard) [reply gapis/service/service.proto]: Indeed! fixed.

**Pull request merged**